awx/main/migrations/0078_v360_clear_sessions_tokens_jt.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-05-08 14:51
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from awx.main.migrations._create_system_jobs import create_clearsessions_jt, create_cleartokens_jt
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0077_v360_add_default_orderings'),
+    ]
+
+    operations = [
+        # Schedule Analytics System Job Template
+        migrations.RunPython(create_clearsessions_jt, migrations.RunPython.noop),
+        migrations.RunPython(create_cleartokens_jt, migrations.RunPython.noop),
+        migrations.AlterField(
+            model_name='systemjob',
+            name='job_type',
+            field=models.CharField(blank=True, choices=[('cleanup_jobs', 'Remove jobs older than a certain number of days'), ('cleanup_activitystream', 'Remove activity stream entries older than a certain number of days'), ('cleanup_facts', 'Purge and/or reduce the granularity of system tracking data'), ('clearsessions', 'Removes expired browser sessions from the database'), ('cleartokens', 'Removes expired OAuth 2 access tokens and refresh tokens')], default='', max_length=32),
```

Comment:
We should take this opportunity to get rid of `cleanup_facts`; we removed it here: https://github.com/ansible/awx/pull/3667/files